core/src/main/codegen/templates/Parser.jj

```diff
@@ -5554,9 +5558,33 @@ SqlNode BuiltinFunctionCall() :
         }
         (
             <USING> name = SimpleIdentifier() {
+<#if parser.allowTranslateUsingCharSetWithError>
+                String nameString = name.toString().toUpperCase();
+                String[] charSets = new String[2];
+                if(nameString.contains("_TO_") && nameString.split("_TO_").length == 2){
```

Comment:
nit: Rather than doing nameString.split() up here and in the body of the if, can we do it in one location and store the result?

Reply:
Fix in https://github.com/googleinterns/calcite/pull/84/commits/8f1e577ebd71a0824d776563241bad47007558e9